deploy/crds/apps.open-cluster-management.io_gitopsclusters.yaml

```diff
@@ -56,26 +60,26 @@ spec:
                 description: "ObjectReference contains enough information to let you
                   inspect or modify the referred object. --- New uses of this type
                   are discouraged because of difficulty describing its usage when
-                  embedded in APIs. 1. Ignored fields.  It includes many fields which
+                  embedded in APIs.  1. Ignored fields.  It includes many fields which
```

Comment:
```suggestion
                      You might have to deal with some of the following difficulties: 
               1. `Ignored fields`.  It includes many fields which are not
                generally valid.  For instance, ResourceVersion and FieldPath are
                both rarely valid in actual usage.  
                2. `Invalid usage help`. You cannot add specific help for individual usage.  In most embedded usages, you have particular restrictions, such as: `\"must refer only to types A and B\"` or ` \"UID not honored\"`  or ` \"name must
 be restricted\"`. You cannot describe these restrictions when they are embedded. 
                3. `Inconsistent validation`.  Because the usages are different, the
                validation rules are different by usage, making it difficult for you to predict what happens. 
                 4. The fields are both imprecise and overly precise.  `Kind` is not a precise mapping to a URL making the direction ambiguous, therefore, possibly requiring a REST mapping. In most cases, the dependency is on the `group,resource` setting and the version of the actual structure is irrelevant.  
                5. We cannot easily change the usage.  Because this type is embedded in many locations, updates to this type affects various schemas.  Therefore, avoid making new APIs and embedding an underspecified API type that they do not control. Instead, create a locally provided and used type that is focused on your reference. For example,  use`ServiceReferences` for admission registration by going to the following link: https://github.com/kubernetes/api/blob/release-1.17/admissionregistration/v1/types.go#L533
```